src/Form/Type/DataFlowType.php

```diff
@@ -37,6 +39,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                         'multiple' => true,
                         'expanded' => true,
                     ])
+                    ->add('schedule', TextType::class)
```

Comment:
Should we use `CronExpressionType::class` here?

Reply:
We could do, but it needs some customization as it's contains a lot of fields